src/rviz_visual_tools.cpp

```diff
@@ -896,6 +902,10 @@ bool RvizVisualTools::publishMarkers(visualization_msgs::MarkerArray &markers)
   if (markers.markers.empty())
     return false;
 
+  if (frame_locking_enabled_)
+    for (auto &marker: markers.markers)
+      marker.frame_locked = true;
```

Comment:
Why is the for loop necessary when all markers should already have this property enabled above where you have:

> marker.frame_locked = frame_locking_enabled_;

?